paddle/math/RowBuffer.h

```diff
@@ -60,7 +60,9 @@ class RowBuffer {
    */
   inline real* get(int row) const {
     if (preallocatedBuf_) {
-      CHECK_LE((row + 1) * width_ * sizeof(real), preallocatedBuf_->getSize());
+      // CHECK_LE((row + 1) * width_ * sizeof(real),
+      // preallocatedBuf_->getSize());
+      CHECK_LE((row)*width_ * sizeof(real), preallocatedBuf_->getSize());
```

Comment:
I am not sure what are these lines for? are these lines accidentally added?

Reply:
Sorry, that's for my local testing, will revert this file.